decentraland/gate
# feat: better transaction lifecycle

This PR uses `decentraland-eth@5.0.0` and `decentraland-dapps@3.1.2` to improve the status of the transactions.

How to test:

1. Send a transaction with a low fee (0.1 Gwei) -> You should see a `Pending` transaction
2. Send N more transactions with high fee (20 Gwei) -> You should see them appear as `Queued`
3. Go to MetaMask and retry the first transaction with a high fee (20 Gwei) -> The pending transaction should change to `Dropped`.

Eventually the replacement transaction will get mined (you will get a notification by MetaMask) and you should see the `Queued` transactions change to `Pending` and then `Confirmed` in the order they were sent as they get mined. Eventually the `Dropped` transaction should also change to `Replaced` and the link should point to the replacement transaction on Etherscan (this may take a little longer than the other transactions getting mined).

Here's an example: [[VIDEO]](https://my.mixtape.moe/cexxls.mp4)
